huggingface/pytorch/inference/docker/build_artifacts/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We keep the `build_artifacts` at the following location: https://github.com/aws/deep-learning-containers/tree/master/huggingface/build_artifacts/inference, will it be possible to shift the artifacts in this PR to the original location where we generally keep the artifacts?

Reply:
Sure, just replaced the artifacts.